packages/ombits-client/src/components/header.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useCallback } from 'react';
+import { useCurrentRoute } from 'react-navi';
+
+import { Nav } from './nav';
+
+export const Header = () => {
+  const route = useCurrentRoute();
+  const { navTabText } = route.state;
+
+  const [isNavVisible, setNavVisibility] = useState(false);
+  const closeNav = useCallback(() => {
```

Comment:
@remotealex we are using callback just to make sure we have memoized this function value

Reply:
yes